docs/reference/manifest/filename.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FileName element in the manifest file
+description: ''
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# FileName element
+
+Specifies the file name of the equivalent XLL for custom functions in your web add-in.
+
+**Add-in type:** Custom function
+
+## Syntax
+
+```XML
+    <FileName>{filename}</FileName>  
+```
+
+## Contained in
+
+[EquivalentAdd-in](equivalentaddin.md)
+
+## Remarks
+
+You must specify the filename of the XLL that contains equivalent user-defined functions (UDF) for your custom functions in your web add-in.
```

Comment:
Isn't the user taking this action here in the manifest?

Reply:
@AlexJerabek  Correct, is something missing here?